src/meta/meta_apply.c

```diff
@@ -20,17 +20,28 @@ __meta_btree_apply(WT_SESSION_IMPL *session, WT_CURSOR *cursor,
     const char *cfg[])
 {
 	WT_DECL_RET;
+	int t_ret;
 	const char *uri;
 	bool skip;
 
+	/*
+	 * Accumulate errors from the applied function(s) but continue through
+	 * to the end of the metadata.
+	 */
+	t_ret = 0;
+
```

Comment:
I wonder if we can restructure this loop so that TRET macros can be used instead of hand coding that (and losing the nice priority of errors that TRET maintains).  Maybe use `c_ret = cursor->next(cursor)`?